catroid/src/main/java/org/catrobat/catroid/stage/StageActivity.java

```diff
@@ -415,4 +419,31 @@ public void addToRequiresPermissionTaskList(RequiresPermissionTask task) {
 	public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
 		permissionRequestActivityExtension.onRequestPermissionsResult(this, requestCode, permissions, grantResults);
 	}
+
+	public static void handlePlayButton(View view, final Activity activity) {
```

Comment:
the parameter `view` is not used in this method, please remove it. the `View view` parameter in the methods in the other Activities is just necessary because it is called from the `.xml`, so it's fine there, but unnecessary here.

But as a suggestion: you could add the `ProjectManager` as a parameter to the method, because then the `ProjectManager.getInstance()` call could happen from the calling activity (e.g. `SpriteActivity`), so that one day, we might be able to refactor the God Object `ProjectManager` so that e.g. testing becomes easier.

Reply:
done